views/annotation/annotationoptions.html

```diff
@@ -55,20 +68,22 @@
               ng-change="ctrl.updateAnnotation()" ng-disabled="!ctrl.annotationOptions.show"
               ng-model="ctrl.annotationOptions.showDescription"
             />
-            <label class="custom-control-label" ng-attr-for="showAnnotationDescription{{ctrl.uid}}">
+            <label class="custom-control-label" ng-attr-for="showAnnotationDescription{{ctrl.uid}}" ng-disabled="disabled">
```

Comment:
I don't think `disabled` is ever set on the scope, nor do `label` elements have a disabled state. Is this needed?

Reply:
I'm not sure what this actually does. I was following the example set somewhere else in the code. It seems to be automatically set so that when the element in `ng-attr-for` is disabled the label also gets disabled.

Reply:
I suspect the other places in this template using `ng-disabled="disabled"` were erroneously copied from another file. The `ngDisabled` directive sets `disabled="disabled"` on an element if the expression is truthy. In this case the expression is watching `$scope.disabled`, which is never set so the expression is always false.

Reply:
Extra tags have been removed, and the color picker inputs are properly disabled when the background is not shown.